src/params/improveMode/ImproveMode.ts

```diff
@@ -0,0 +1,2 @@
+export const OUTDOOR = 'outdoor';
```

Comment:
Should this be under constants rather than params?

Reply:
I'm not sure it makes sense to keep both Constants and Params, one of them has to go (And we also need a better name).

I think everything that comes from the "outside" needs to go Params